starter-code/src/viking.js

```diff
@@ -1,11 +1,62 @@
 // Soldier
-function Soldier () {}
+function Soldier (health, strength) {
+  this.health = health;
+  this.strength = strength;
+  this.attack = function(){
```

Comment:
`attack` and `receiveDamage` should be methods of `Soldier.prototype` instead of each soldier object individually.